src/prefect/tasks/firebolt/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+This module contains a collection of tasks for interacting with firebolt databases via
+the firebolt-sdk-python library.
```

Comment:
I think this should be "firebolt-python-sdk" per https://github.com/firebolt-db/firebolt-python-sdk